Sources/AppStoreConnectCLI/Commands/TestFlight/BetaTesters/InviteBetaTesterCommand.swift

```diff
@@ -29,20 +26,23 @@ struct InviteBetaTesterCommand: CommonParsableCommand {
             help: "Names of TestFlight beta tester group that the tester will be assigned to.")
     var groups: [String]
 
+    func validate() throws {
+        if groups.isEmpty {
+            throw ValidationError("Invalid input, you must provide at least one group name.")
```

Comment:
If group name is mandatory shouldn't it be declared as an argument and not as an option?

Reply:
If marked as `Argument` we will not be able to do `--group-names` in input

Reply:
If you are throwing error when the optional groups field is empty then it should be an argument because as per your logic you won't proceed if the user hasn't entered a value for the groups..using @option means its optional